docs/guide.rst

```diff
@@ -320,9 +269,354 @@ attributes ``description`` and ``link``::
+
+In the previous chapter we've learned what :ref:`/spec/tests`,
+:ref:`/spec/plans` and :ref:`/spec/stories` are used for. Now the
+time has come to learn how to efficiently organize them in your
+repository. First we'll describe how to easily :ref:`create` new
+tests, plans and stories, how to use :ref:`lint` to verify that
+all metadata have correct syntax. Finally, we'll dive into
+:ref:`inheritance` and :ref:`elasticity` which can substantially
+help you to minimize data duplication.
+
+.. _create:
+
+Create
+------------------------------------------------------------------
+
+When working on the test coverage, one of the most common actions
+is creating new tests. Use ``tmt test create`` to simply create a
+new test based on a template::
+
+    $ tmt test create /tests/smoke
+    Template (shell or beakerlib): shell
+    Directory '/home/psss/git/tmt/tests/smoke' created.
+    Test metadata '/home/psss/git/tmt/tests/smoke/main.fmf' created.
+    Test script '/home/psss/git/tmt/tests/smoke/test.sh' created.
+
+As for now there are two templates available, ``shell`` for simple
+scripts written in shell and ``beakerlib`` with a basic skeleton
+demonstrating essential functions of this shell-level testing
+framework. If you want to be faster, specify the desired template
+directly on the command line using ``-t`` or ``--template``::
+
+    $ tmt test create --template shell /tests/smoke
+    $ tmt test create --t beakerlib /tests/smoke
+
+In a similar way, the ``tmt plan create`` command can be used to
+create a new plan with templates::
+
+    tmt plans create --template mini /plans/smoke
+    tmt plans create --t full /plans/features
+
+When creating many plans, for example when migrating the whole
+test coverage from a different tooling, it might be handy to
+override default template content directly from the command line.
+For this use individual step options such as ``--discover`` and
+provide desired data in the ``yaml`` format::
+
+    tmt plan create /plans/custom --template mini \
+        --discover '{how: "fmf", name: "internal", url: "https://internal/repo"}' \
+        --discover '{how: "fmf", name: "external", url: "https://external/repo"}'
+
+Now it will be no surprise for you that for creating a new story
+the ``tmt story create`` command can be used with the very same
+possibility to choose the right template::
+
+    tmt story create --template full /stories/usability
+
+Sometimes you forget something, or just things may go wrong and
+you need another try. In such case add ``-f`` or ``--force`` to
+quickly overwrite existing files with the right content.
+
+.. _lint:
+
+Lint
+------------------------------------------------------------------
+
+It is easy to introduce a syntax error to one of the fmf files and
+make the whole tree broken. The ``tmt lint`` command performs a
+set of :ref:`lint-checks` which compare the stored metadata
+against the specification and reports anything suspicious:
+
+.. code-block:: shell
```

Comment:
For blocks like this, I'd recommend `console` instead of `shell`. Depending on the syntax highlighting backend, results may vary, but in general, it would focus on highlighting commands and other command-line elements rather than tokens allowed in a shell script. And it's semantically more fitting, as the block below demonstrates something happening on a console, after typing a command.

Reply:
I tried the `console` version before but it seemed to me that it does not render reasonably. For example, comments do not seem to be respected:

> **#** Change the **set** of checks applied - **enable** some, disable others
> tmt lint --enable-check T001 --disable-check C002

Here only the `#` char is displayed in a different color and thus random (shell) words are highlighted in the descriptions.

Reply:
![obrazek](https://github.com/teemtee/tmt/assets/2803150/cb450da0-80ef-4e62-98d3-9ffc1eb2bdc6)

Reply:
Agreed on chat that `shell` seems to give the best output from the current available options. Polishing will come later.